packages/suite/src/utils/suite/device.ts

```diff
@@ -121,6 +121,12 @@ export const getFwVersion = (device: AcquiredDevice) => {
     return `${features.major_version}.${features.minor_version}.${features.patch_version}`;
 };
 
+export const getFwRevision = (device: AcquiredDevice) => {
+    const { features } = device;
+    const rev = Buffer.from(features.revision, 'hex');
```

Comment:
We are currently looking to reduce the amount of polyfills required in Suite and `Buffer` is one of them. There might maybe be some utils in Suite to convert from hex to string but I can't remember where (if it exists) from the top of my head. If it doesn't exist, it should be fairly trivial to implement.

Reply:
tbh. i don't think we will ever get rid of `Buffer` polyfil, it's used in so many places and dependecies...

Reply:
I'd love to reduce the amount of polyfills we have in the project, especially given the alternatives. Not allowing new uses of `Buffer` in incoming changes is a good step towards that direction.

Reply:
whats your plan to reduce usage of `Buffer` in suite dependencies? 
like crypto/coins libs (btc, eth, xrp...), address validators libs, etc?

Reply:
Hows something like this:

```
export const getFwRevision = (device: AcquiredDevice) => hex2String(device.features.revision)

export const hex2String = (hex: string) => {
    let str = '';
    for (let i = 0; i < hex.length; i += 2) {
        str += String.fromCharCode(parseInt(hex.substr(i, 2), 16));
    }
    return str;
};
```

Reply:
14d766f

Reply:
@szymonlesisz I still have to evaluate how widespread the usage of Node APIs (which are polyfilled for the front-end) is. I'm aware Buffer is useful in our project but there are ways to do most of it (if not all) using APIs available in the browser. Not sure how long that's gonna take but I wouldn't want to introduce new code that relies on this polyfill in the project.

@waltonzt Looks good, but I would suggest to move the `hex2String` function in a different utils file such as `suite/string.ts` as it is quite generic.

Reply:
i understand the intention and i have nothing against reducing polyfils, yet i don't understand how you want deal with that regarding dependencies while all essential crypto modules are using `Buffer`